src/main/jbake/content/jms-examples003.adoc

```diff
@@ -31,10 +31,10 @@ Delivery]
 
 [[CHDCEFGA]][[overview-of-writing-simple-jms-application]]
```

Comment:
overview-of-writing-simple-jakarta-messaging-application

(You get the idea, I'm not going to keep marking these.)